### PR TITLE
RA-1191

### PR DESCRIPTION
**Pull request by SandraPG85**
Add circular statistics behavior

### CODE REVIEW

File: VAS.Core/ViewModel/Statistics/PercentCircularChartVM.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+{
+	/// <summary>
+	/// View model for specific charts where a percentage is displayed in 
+	/// the middle of a circle
+	/// </summary>
+	public sealed class PercentCircularChartVM : CircularChartVM
+	{
+		public PercentCircularChartVM (SeriesVM serie, int total)
+		{
+			SeriesVM emptySerie = new SeriesVM ("empty", total - serie.Elements, App.Current.Style.ChartBase);
```
Comment (vguzmanp) [inline VAS.Core/ViewModel/Statistics/PercentCircularChartVM.cs]: Empty should be transparent by default, shouldn't it?

File: VAS.Core/ViewModel/Statistics/CircularChartVM.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+		/// </summary>
+		/// <value>The circle radius.</value>
+		public int Radius { get; set; }
+
+		/// <summary>
+		/// Gets or sets the thickness of the arc line
+		/// </summary>
+		/// <value>The thickness.</value>
+		public int LineWidth { get; set; }
+	}
```
[the remark this answers is not in the document]
Comment (SandraPG85) [reply VAS.Core/ViewModel/Statistics/CircularChartVM.cs]: moved to ChartVM

File: VAS.Core/ViewModel/Statistics/PercentCircularChartVM.cs (hunk 1 above)
Comment (SandraPG85) [reply VAS.Core/ViewModel/Statistics/PercentCircularChartVM.cs]: mm i don't think so, then the graphic never will be circular ;)

File: VAS.Drawing/Canvas.cs [hunk 3]
```diff
@@ -282,7 +282,9 @@ protected void End ()
 
 		protected void DrawBackground ()
 		{
-			tk.Clear (BackgroundColor);
+			if (BackgroundColor != Color.Transparent) {
+				tk.Clear (BackgroundColor);
```
[the remark this answers is not in the document]
Comment (vguzmanp) [reply VAS.Drawing/Canvas.cs]: Add FIXME, or whatever

**Pull request merged**